.github/workflows/ci-trilinos.yml

```diff
@@ -67,7 +66,7 @@ jobs:
           -D EIGEN_INCLUDE_DIR=/eigen/eigen-${{ env.eigen_version }} \
           -D Trilinos_DIR=${{ env.trilinos_dir }}/lib/cmake/Trilinos \
           -D CMAKE_INSTALL_PREFIX:PATH=../install \
-          -D CMAKE_CXX_FLAGS='-Wall -Werror'
```

Comment:
This is necessary because Trilinos has deprecated Epetra, resulting in a bunch of warnings:

```
#warning "The Epetra package is deprecated"
```